<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>S-DIA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #e9ecef;
}
QLabel {
    color: #000000;
    font-size: 14px;
    background-color: transparent;
}
QLineEdit {
    padding: 8px 12px;
    border: 1px solid #ced4da;
    border-radius: 4px;
    background-color: #ffffff;
    font-size: 14px;
    min-height: 20px;
    color: #495057;
}
QLineEdit:focus {
    border: 1px solid #80bdff;
    border-color: #80bdff;
    outline: 0;
}
QPushButton#login_button {
    color: #fff;
    background-color: #007bff;
    border: 1px solid #007bff;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
    font-weight: bold;
}
QPushButton#login_button:hover {
    color: #fff;
    background-color: #0069d9;
    border-color: #0062cc;
}
QPushButton#login_button:pressed {
    background-color: #0062cc;
    border-color: #005cbf;
}
QPushButton {
    border: 1px solid #ced4da;
}
QCheckBox {
    color: #212529;
    font-size: 14px;
}
QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 1px solid #ced4da;
    border-radius: 3px;
    background-color: white;
}
QCheckBox::indicator:checked {
    background-color: #007bff;
    border-color: #007bff;
}
QFrame#form_container {
    background-color: white;
    border-radius: 8px;
    border: 1px solid #dee2e6;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>40</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>40</number>
    </property>
    <item>
     <widget class="QWidget" name="title_container" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="title_label">
         <property name="styleSheet">
          <string notr="true">font-size: 32px;
font-weight: bold;
color: #000000;</string>
         </property>
         <property name="text">
          <string>S-DIA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="version_label">
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-weight: normal;
color: #6c757d;
padding-left: 8px;</string>
         </property>
         <property name="text">
          <string>v1.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="release_label">
         <property name="styleSheet">
          <string notr="true">font-size: 12px;
font-weight: normal;
color: #6c757d;
padding-left: 4px;</string>
         </property>
         <property name="text">
          <string>r202504_1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="form_container">
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: white;
    border-radius: 8px;
    border: 1px solid #dee2e6;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <property name="bottomMargin">
        <number>24</number>
       </property>
       <item>
        <widget class="QLabel" name="login_title">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 20px;
font-weight: bold;
color: #000000;
margin-bottom: 8px;
border: 0;</string>
         </property>
         <property name="text">
          <string>로그인</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="id_layout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLineEdit" name="id_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>아이디를 입력하세요</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="pw_layout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLineEdit" name="pw_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
           <property name="placeholderText">
            <string>비밀번호를 입력하세요</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bottom_layout">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QCheckBox" name="save_id_checkbox">
           <property name="styleSheet">
            <string notr="true">color: #212529;
font-size: 14px;
font-weight: normal;</string>
           </property>
           <property name="text">
            <string>아이디 저장</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="login_button">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>로그인</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="copyright_label">
      <property name="styleSheet">
       <string notr="true">color: #6c757d;
font-size: 12px;
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>© 유피에스데이터 (UPSDATA).</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
